schemas/tasks/definitions.yaml

```diff
@@ -1,10 +1,10 @@
 task_id:
   type: string
-  pattern: ^[0-9a-z-_]+$
+  pattern: ^[0-9a-z_-]+$
   description: The identifier for the Task.
   example: profile_123
 task_def_id:
   type: string
-  pattern: ^[0-9a-z-_]+$
+  pattern: ^[0-9a-z_-]+$
```

Comment:
This is to avoid below warning when running ruby integration tests:
`warning: character class has '-' without escape: /^[0-9a-z-_]+$/`